project_description/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Project Description
+===================
+
+This module extends the functionality of projects to support setting a
+description for each project and allow you to search by that description.
+
+Usage
+=====
+
+To set a description for a project, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Edit or create one.
+#. You will see a new *Description* tab, with an HTML area to set the project's
+  description.
```

Comment:
This line causes the runbot failure. Write only one single line for each item of the list.

Reply:
Or you can indent this line with 3 spaces at the beginning